lib/spawn-normal.js

```diff
@@ -1,10 +1,15 @@
 
 var spawn = require('child_process').spawn
+var _ = require('lodash')
 
 module.exports = function (command, args, options, done) {
+  // Prevent https://github.com/joyent/node/issues/9158
+  options = _.extend({}, options)
+
+  // Trap any other synchronous errors
   try {
     done(null, spawn(command, args, options))
-  } catch (e) {
-    done(127)
```

Comment:
A shell exit code isn't expected here, since no command has been run. Should just pass the error as usual.